docs/tutorials/get_started.ipynb

```diff
@@ -0,0 +1,247 @@
+      ],
+      "text/plain": [
+       "<Figure size 700x500 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.visualization import plot_histogram\n",
+    "\n",
+    "plot_histogram(\n",
+    "    job.result().quasi_dists\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77054d4e",
+   "metadata": {},
+   "source": [
+    "And that's it! You now have everything you need to write and run quantum programs."
```

Comment:
```suggestion
    "And that's it! You now have everything you need to get going writing and running quantum programs."
```
A little less of a bold claim

Reply:
How about "to start writing and running quantum programs"? My friends who are English language learners hate phrasal verbs and I suspect that "get going" might be less accessible.

Reply:
Quick case for the original: It's more direct, optimistic, and I don't think it's untrue?